app/routes/event.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const auth = require('../middleware/auth')
+const router = express.Router()
+const eventController = require('../controllers/event')
+
+// CREATE A POST
+router.post(
+  '/',
+  auth,
+  eventController.createEvent
+)
+
+router.post(
+  '/event_update/:id',
+  auth,
+  eventController.updateEvent
+)
+router.post(
+  '/:id/event_rsvp/:event_id',
```

Comment:
`/:id/event_rsvp/:event_id` in this route:
what is - 
1. `/:id`?
2. `/:event_id`?
3. why do you want `/event_rsvp`?

Reply:
there could be various events, so we need to have clearance which user(:id) has rsvped to which event(:event_id)

Reply:
we dont need user it to be part of URL param.
it will come in the payload.
it should be just `/:eventId`
and then the payload going to server will have the data which user done this action.